### PR TITLE
[MRG] Add Contributing guide 

### PR DESCRIPTION
**Pull request by massich**
works on #5281 by adding a contributing.md according to [this](https://help.github.com/articles/setting-guidelines-for-repository-contributors/)

### CODE REVIEW

**Comment by massich** [comment]
do you guys think we should do much more that that?

**Comment by massich** [comment]
I think that he only important part is that it actually appears when a newcomer tries to do an issue or PR

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Contributing to MNE-Python
+
+First off, thanks for taking the time to contribute!
+
+The following is quick summary to a set of guidelines for contributing to [MNE-Python](https://github.com/mne-python/mne-python) on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
```
Comment (agramfort) [inline .github/CONTRIBUTING.md]: is -> is a

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Contributing to MNE-Python
+
+First off, thanks for taking the time to contribute!
+
+The following is a quick summary to a set of guidelines for contributing to [MNE-Python](https://github.com/mne-python/mne-python) on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+#### Table Of Contents
```
Comment (cbrnr) [inline .github/CONTRIBUTING.md]: I think we can skip the TOC in such a short document.

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+[Code of Conduct](#code-of-conduct)
+
+[How Can I Contribute?](#how-can-i-contribute)
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [MNE-Python's Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [mne-conduct@googlegroups.com](mailto:mne-conduct@googlegroups.com).
+
+## How can I contribute
```
Comment (cbrnr) [inline .github/CONTRIBUTING.md]: Add a question mark here.

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+Before contributing make sure that you read [our contributing guide](https://martinos.org/mne/contributing.html)
```
Comment (cbrnr) [inline .github/CONTRIBUTING.md]: ...make sure you are familiar with out contributing guide...

Missing period.

File: .github/CONTRIBUTING.md (hunk 3 above)
Comment (agramfort) [reply .github/CONTRIBUTING.md]: How to contribute

then

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (larsoner) [reply .github/CONTRIBUTING.md]: agreed

**Comment by massich** [comment]
> Shouldn't it be in the root folder?

From github's documentation:

> Decide whether to store your contributing guidelines in your repository's root, `docs`, or `.github` directory. 

IMHO, since this is only a reference to `doc/contributing.rst` to keep github happy, to me it makes sense to put it there. (And add, `.github/pull-request.md` and `issue.md`)

Otherwise, whe should find the manner of having `doc/contributing.md` and `doc/contributing.rst` import one from another and keep the content in a single place.

**Comment by cbrnr** [comment]
Most projects seem to have it in their roots, and most people will go looking for it there. Even if it is just  a pointer to the real doc, I'd put it in root so people can access it outside of GitHub's use cases.

**Comment by larsoner** [comment]
@cbrnr feel free to merge if you are happy

**Comment by cbrnr** [comment]
@massich can you fix the error in the first Travis job? You need to add the file to MANIFEST.in.

**Comment by cbrnr** [comment]
I tried to push this change but I don't seem to have access? @larsoner I know you've done this before, how can I push to forks from other people? Here's what I tried:

```
git fetch massich
git checkout -t massich/add_contributing_guide
[Make the change]
git commit -a -m "Add to MANIFEST.in"
git push
```

But then it says authentication failed.

**Comment by agramfort** [comment]
Use https and not git ssh protocol

**Comment by larsoner** [comment]
I always push with `git ssh`, even to branches of other people. It's possible that for some reason you can't push directly. If so, feel free to make a commit, push a branch to your fork, and @massich or I can cherry-pick it into his branch.

**Comment by cbrnr** [comment]
Thanks @larsoner. This seems like too much effort for this tiny change, can you or @massich just add `CONTRIBUTING.md` to `MANIFEST.in`?

**Comment by larsoner** [comment]
Looks like he did an hour or so ago

**Comment by cbrnr** [comment]
Ah, didn't see that. Thanks @massich!

**Pull request merged**